tests/_utils/cleanup.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/* global document */
+
+/**
+ * Removes all the `.ck-body` elements available in the DOM.
+ *
+ * It is commonly used to cleanup after editors that test editor crashes.
+ *
+ * See https://github.com/ckeditor/ckeditor5/issues/6018 for more details.
+ */
+export function removeEditorBodyOrphans() {
```

Comment:
I'm not sure we need these `removeEditorBodyOrphans()` and `destroyEditorOrphans()` at all. The first is called once in ckeditor5-core and the latter is called twice in ckeditor5-watchdog. Since they are not required anywhere else, they should be local to the tests that use them.

Reply:
You're right about `destroyEditorOrphans`, however `removeEditorBodyOrphans` method is also reused in `editor-*` packages. I believe that having `destroyEditorOrphans` (in `utils/cleanup`) give it some form of discoverability, inlining it in usage place it will get duplicated if someone needs something like that.

@oleq WDYT in this case?

Reply:
Alright, that makes sense. I didn't notice `removeEditorBodyOrphans` in `ckeditor5-editor-` because my branches were outdated. Let's ditch `destroyEditorOrphans` only then. 

Let's also have a test for `removeEditorBodyOrphans` maybe?

Reply:
Function `destroyEditorOrphans` was inlined in the watchdog package, while test for `removeEditorBodyOrphans` was added in 4e3c55e.